src/main/java/uk/gov/ch/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.ch.exception;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(ConstraintViolationException.class)
+    public ResponseEntity<Object> handleInvalidOverseasEntityNumberException(ConstraintViolationException ex) {
```

Comment:
Suggest renaming this method to just 'handleConstraintViolationException' as it will in theory be called for any unhandled runtime ConstraintViolationExceptions.